library/kotlin/test/io/envoyproxy/envoymobile/RequestMapperTest.kt

```diff
@@ -123,7 +145,7 @@ class RequestMapperTest {
         perRetryTimeoutMS = 9001)
     val retryPolicyHeaders = retryPolicy.outboundHeaders()
 
-    val requestHeaders = RequestBuilder(method = RequestMethod.POST, scheme = "https", authority = "api.foo.com", path = "/foo")
+    val requestHeaders = RequestBuilder(method = RequestMethod.POST, scheme = "https", authority = "api.foo.com", path = "/foo", upstreamHttpProtocol = UpstreamHttpProtocol.HTTP2)
```

Comment:
Changing this to a builder as @goaway suggested would allow for reverting a lot of these test changes

Reply:
Yep, I like the suggestion to move out to the builder. It would also make it easier to deprecate if we want to move into an only ALPN-based future when that is supported.